opentracing-api/src/main/java/io/opentracing/SpanContext.java

```diff
@@ -26,6 +26,38 @@
  * @see Span#getBaggageItem(String)
  */
 public interface SpanContext {
+    /**
+     * Return the ID of the trace.
+     *
+     * Should be globally unique. Every span in a trace shares this ID.
+     *
+     * An empty String will be returned if the tracer does not support this functionality
+     * (this is the case for no-op tracers, for example). null is an invalid return value.
+     *
+     * Observe there is no restriction regarding the format of the returned value, as this
+     * should simply be the equivalent of calling toString() on the native ID object
+     * used by the tracer.
```

Comment:
This wording is problematic. It says "should ... ToString" which in many cases is the wrong thing to do, e.g. for int64 IDs. If we want to make a recommendation, it would be that the string format SHOULD match the representation of the IDs in http headers.

The javadoc here should ideally be repeating word for word the specification.

Reply:
Oh, good point and good example. So I feel that we should remove this clarification then (if later we add something in the specification itself, we can add it here). Would that make sense to you?

Reply:
Yes, can remove it altogether